.travis.yml

```diff
@@ -39,8 +53,20 @@ script:
 after_failure:
   cat v6_api/nohup.out
```

Comment:
Just suprised to see a reference to an API file here.
But perhaps it's added only for travis to "simulate" the API calls?
Is this command correct?